src/pages/Admin/Teams/NewTeam.vue

```diff
@@ -0,0 +1,106 @@
+<script>
+import { mapGetters } from 'vuex'
+
+import InviteUsers from '@/pages/Admin/Teams/InviteUsers'
+import TeamForm from '@/pages/Admin/Teams/TeamForm'
+
+export default {
+  components: { TeamForm, InviteUsers },
+  data() {
+    return {
+      step: 1
+    }
+  },
+  computed: {
+    ...mapGetters('license', ['permissions', 'license', 'planType']),
+    multitenancy() {
+      return this.license?.terms?.tenants > 1
+    }
+  },
+  watch: {},
+  methods: {}
```

Comment:
can we remove this?

Reply:
Sure! Also looks like I have a lot of conflicts to resolve